forge-game/src/main/java/forge/game/trigger/TriggerBecomesCrewed.java

```diff
@@ -27,6 +29,17 @@ public boolean performTest(Map<AbilityKey, Object> runParams) {
                 return false;
             }
         }
+        if (hasParam("ValidCrewAmount")) {
```

Comment:
usually good to put new params and related cards in separate PR

Reply:
Yeah, gonna do that on my next PR though since the PR is getting big.